Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -412,6 +408,49 @@
   products:
     SodiumCarbonate: 4
 
+- type: reaction
+  id: Benzene
+  impact: Medium
```

Comment:
If this is a normal precursor, then it should not have a "Medium" impact.

Reply:
I just added that because it could be used as a poison